.teamcity/Gradle_Check/model/CIBuildModel.kt

```diff
@@ -35,29 +35,29 @@ data class CIBuildModel (
                             SpecificBuild.SmokeTests),
                     functionalTests = listOf(
-                            TestCoverage(TestType.platform, OS.linux, JvmVersion.java7),
-                            TestCoverage(TestType.platform, OS.windows, JvmVersion.java8)),
+                            TestCoverage(TestType.platform, OS.linux, JvmVersion.java8),
+                            TestCoverage(TestType.platform, OS.windows, JvmVersion.java10)),
                     performanceTests = listOf(PerformanceTestType.test),
                     omitsSlowProjects = true),
             Stage("Master Accept", "Rerun tests in different environments / 3rd party components",
                     trigger = Trigger.eachCommit,
                     functionalTests = listOf(
-                            TestCoverage(TestType.quickFeedbackCrossVersion, OS.linux, JvmVersion.java7),
-                            TestCoverage(TestType.quickFeedbackCrossVersion, OS.windows, JvmVersion.java7),
+                            TestCoverage(TestType.quickFeedbackCrossVersion, OS.linux, JvmVersion.java8),
+                            TestCoverage(TestType.quickFeedbackCrossVersion, OS.windows, JvmVersion.java8),
                             TestCoverage(TestType.platform, OS.linux, JvmVersion.java10),
-                            TestCoverage(TestType.parallel, OS.linux, JvmVersion.java7, JvmVersion.java8, JvmVendor.ibm))
+                            TestCoverage(TestType.parallel, OS.linux, JvmVersion.java8, JvmVersion.java10, JvmVendor.ibm))
             ),
             Stage("Release Accept", "Once a day: Rerun tests in more environments",
                     trigger = Trigger.daily,
                     functionalTests = listOf(
-                            TestCoverage(TestType.soak, OS.linux, JvmVersion.java8),
+                            TestCoverage(TestType.soak, OS.linux, JvmVersion.java10),
                             TestCoverage(TestType.soak, OS.windows, JvmVersion.java8),
-                            TestCoverage(TestType.allVersionsCrossVersion, OS.linux, JvmVersion.java7),
-                            TestCoverage(TestType.allVersionsCrossVersion, OS.windows, JvmVersion.java7),
+                            TestCoverage(TestType.allVersionsCrossVersion, OS.linux, JvmVersion.java8),
+                            TestCoverage(TestType.allVersionsCrossVersion, OS.windows, JvmVersion.java8),
                             TestCoverage(TestType.noDaemon, OS.linux, JvmVersion.java8),
-                            TestCoverage(TestType.noDaemon, OS.windows, JvmVersion.java8),
+                            TestCoverage(TestType.noDaemon, OS.windows, JvmVersion.java10),
                             TestCoverage(TestType.platform, OS.macos, JvmVersion.java8),
-                            TestCoverage(TestType.platform, OS.linux, JvmVersion.java9)),
+                            TestCoverage(TestType.platform, OS.linux, JvmVersion.java10)),
```

Comment:
Well, I'm not aware of the Oracle recommended way, but I still think it's necessary to keep at least one set of Java9 test so that we can find out potential Java9-specific issues quickly. @wolfs WDYT?

Reply:
I am fine with it either way. Maybe we should have at least one build using Java 9 to see if we don't break anything.